tau-component-packages/components/toggleslider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "label": "toggleslider",
-  "selector": ".ui-toggle-container, .ui-iot-toggleslider, .ui-iot-toggleswitch, .ui-toggleslider-base",
+  "selector": ".ui-toggle-container, .ui-iot-toggleslider, .ui-iot-toggleswitch, .ui-toggleslider-base, .ui-toggle-slider-container",
```

Comment:
We no longer use iot components. I think we can remove the iot related classes from the selector.

Reply:
Done